.devcontainer/Dockerfile

```diff
@@ -31,8 +31,9 @@ RUN apt-get install --assume-yes \
   "git=1:2.*" \
   "zsh=5.*"
 
-COPY "./scripts/*" "$HOME/"
-RUN curl -fsSL "https://github.com/cashapp/hermit/releases/download/v0.26.1/install.sh" | bash
+COPY ./scripts/ /root/
```

Comment:
I wonder what is the issue with `$HOME`? can you please elaborate?
Would it cause issues in pure Docker/outside dev containers, like CI?

Reply:
$HOME isn' available in the docker context at that point it seems. Been bothering me for ages.